src/components/study/Registration.vue

```diff
@@ -25,27 +29,30 @@
 import MainNav from './MainNav.vue'
 import store from '../store'
 import axios from 'axios';
+import { VueTelInput } from 'vue-tel-input'
 
 export default {
   name: 'StudyRegistration',
-  components: { MainNav },
+  components: { MainNav, VueTelInput },
   data() {
     return {
-      phone: '',
-      showAndroid: window.queryParams.android === "true"
+      phoneValue: '',
+      countryCodes: [this.$t('registration-screen.other-region-code'), 'US'],
+      defaultCountry: this.$i18n.locale == 'en' ? 'US' : this.$t('registration-screen.other-region-code'),
+      selectedRegionCode: '',
+      isValidPhoneNumber: false
     }
   },
-  mounted() {
-    var input = this.$refs.phoneField.$refs.input
-    input.type = "tel"
-    input.pattern = "[0-9]*"
-  },
   computed: {
     hasNumber: function() {
-      return this.phone.replace(/\D/g,'').length >= 10 ? 1 : .5
-    }
+      return ( this.phoneValue && this.phoneValue.replace(/\D/g,'').length >= 10 ) ? 1 : .5
+    },
   },
   methods: {
+    onCountryChange: function (countryObject) {
+      this.selectedRegionCode = countryObject.iso2
```

Comment:
Update the local selectedRegionCode with this value (this works, but is this is the right way of doing this in this framework when the external component provides an event handle hook?)

Reply:
I think so, yes. There is an activeCountryCode property in the component but I'm not sure how you'd bind to it, possibly :selectedRegionCode="activeCountryCode"